react/react1/week3/homework/todo-list/src/components/AddTodo.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import Moment from "moment";
+
+export default function AddTodo({ addTodo }) {
+  const [description, setDescription] = useState("");
+  const [deadline, setDeadline] = useState(null);
+
+  const handleDeadline = (date) => {
+    setDeadline(date);
+  };
+
+  const handleDescription = (event) => {
+    setDescription(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
```

Comment:
Nothing wrong with this function but think the code can be improved. This is how I would do it with some comments
`const handleSubmit = (event) => {
    event.preventDefault();

    if (!description || !deadline) {
      alert("Please, add some description and a deadline for your todo!");
      return; // Else not needed because of early return
    } 

    const formattedDeadline = Moment(deadline).format("L");
    addTodo(description, formattedDeadline);
    setDeadline("");
    setDescription("");
    // The line below should not really be needed? 
    // Bind the value of the state with the input instead
    document.getElementById("description").value = "";
}`

!description will return true if it has no length and !deadline will return true if it is null
Think early return in the if statement with the logic you want to do on the bottom is cleaner then a if/else

Let me know if it's not super clear :)